advocacy_docs/partner_docs/QuestFoglightforDatabases/03-SolutionSummary.mdx

```diff
@@ -0,0 +1,11 @@
+---
+title: 'Solution Summary'
+description: 'Brief explanation of the solution and its purpose'
+---
+Quest Foglight for databases is a tool that enables monitoring of health and performance issues for all database platforms, whether on-premise, hybrid, or in the cloud.  This tool enables the user to collect their database information in one place and monitor them simultaneously. It offers 24/7 unattended data collection with a straightforward deployment for monitoring.
+
+DBA's can utilize Quest Foglight for Databases' PostgreSQL monitoring capabilities to detect, diagnose, and resolve performance issues on EDB Postgres Advanced and EDB Postgres Extended database servers.  Quest Foglight for Databases uses PostgreSQL 10.x and above, or EDB Postgres 10.x and above to connect their databases and collect and monitor data.  It is important to monitor your databases for health issues so when an issue occurs, you can diagnose and resolve it quickly.
```

Comment:
"...EDB Postgres 10.x and above..." should be changed to "...EDB Postgres Advanced Server (version and above) and EDB Postgres Extended Server (version and above)..."